src/app/components/form-viewer/form-viewer.component.ts

```diff
@@ -48,25 +55,36 @@ export class FormViewerComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
+    // Results Handling
+    // let results$ = this.refreshTrigger$.pipe(
```

Comment:
delete unused code

Reply:
Left in as placeholder for later implementation.